multiboot2/src/lib.rs

```diff
@@ -389,7 +389,7 @@ impl Reader {
 
     pub(crate) fn read_u8(&mut self) -> u8 {
         self.off += 1;
-        unsafe { core::ptr::read(self.ptr.add(self.off - 1)) }
```

Comment:
oops,  this was committed by accident. But this shouldn't change everything, because dereferencing a u8 or reading it, result both in a copy.